### ISSUE TITLE
zpool wait all - wait on all background process to complete

### ISSUE THREAD

**Issue by Lady-Galadriel** [openzfs/zfs#10114]
### System information

Type | Version/Name
 --- | --- 
Distribution Name	| All
Distribution Version	| All
Linux Kernel	| All
Architecture	| All
ZFS Version	| Master from 2020/03/10
SPL Version	| N/A

### Describe the feature you would like to add
We now have many `zpool wait` items. However, we may want an `all` option in cases were multiple items need to be waited.

### Describe how to use the new feature
Use `zpool wait all` instead of `zpool wait ITEM1` and `zpool wait ITEM2`

A varient could be to allow multiple items on the command line;
```
zpool wait free,scrub POOL
```
but this might be harder to encode. (Unless each wait item were a bit mask entry, then it might be practical.)

### State priority of feature request
I'd say this is a low priority item, (if implemented at all). However, it does round out the feature set of `zpool wait`.

**Comment by ahrens** [comment]
The functionality you're requesting is already implemented (`zpool wait poolname` and `zpool wait -t free,scrub poolname`).  Are you asking for the syntax to be augmented to accept `zpool wait all poolname` and `zpool wait free,scrub poolname`?

```
     zpool wait [-Hp] [-T u|d] [-t activity[,activity]...] pool [interval]
             Waits until all background activity of the given types has ceased
             in the given pool.  
...
             If no activities are specified, the command waits until background
             activity of every type listed below has ceased.
```
